### PR TITLE
Improve MIME Test Coverage

### PR DESCRIPTION
**Pull request by TharmiganK**
## Purpose

Fixes [`ballerina-platform/ballerina-standard-library#1426`](https://github.com/ballerina-platform/ballerina-standard-library/issues/1426)

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -11,3 +11,5 @@ ballerinaGradlePluginVersion=0.8.3
 
 stdlibIoVersion=0.6.0-beta.1
 stdlibLogVersion=1.1.0-beta.1
+
+mockitoVersion=3.7.7
```
Comment (chamil321) [inline gradle.properties:15]: Let's move this entry to  L11

File: mime-ballerina/tests/mime_test.bal [hunk 2]
```diff
@@ -37,6 +37,13 @@ function getDispositionTestObj() returns ContentDisposition {
     return disposition;
 }
 
+function getDispositionWithoutParamsTestObj() returns ContentDisposition {
+    ContentDisposition disposition = new;
+    disposition.disposition = "inline";
+    return disposition;
+}
+
+
```
Comment (chamil321) [inline mime-ballerina/tests/mime_test.bal:46]: Remove this extra new line

File: mime-native/src/test/resources/testng.xml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (chamil321) [inline mime-native/src/test/resources/testng.xml:4]:
```suggestion
  ~ Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: mime-native/src/test/resources/testng.xml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    <test name="Ballerina Mime util Tests" parallel="false">
+        <classes>
+            <class name="org.ballerinalang.mime.util.MimeUtilTest"/>
+            <class name="org.ballerinalang.mime.util.EntityHeaderHandlerTest"/>
+            <class name="org.ballerinalang.mime.util.HeaderUtilTest"/>
+            <class name="org.ballerinalang.mime.util.EntityWrapperTest"/>
+            <class name="org.ballerinalang.mime.util.MultipartDataSourceTest"/>
+        </classes>
+    </test>
+
```
Comment (chamil321) [inline mime-native/src/test/resources/testng.xml:33]:
```suggestion
```

File: mime-native/build.gradle [hunk 5]
```diff
@@ -36,8 +40,38 @@ dependencies {
...
 }
 
+jacocoTestReport {
+    dependsOn test
+    reports {
+        xml.enabled true
+    }
+}
+
+
```
Comment (chamil321) [inline mime-native/build.gradle:74]:
```suggestion
```

File: mime-ballerina/tests/mime_test.bal [hunk 6]
```diff
@@ -973,6 +1005,15 @@ public function testSetContentDispositionToEntity() {
...
 
+@test:Config {}
+public function testSetContentDispositionToEntityWithoutParams() {
+    Entity entity = new;
+    entity.setContentDisposition(getDispositionWithoutParamsTestObj());
+    ContentDisposition disposition = entity.getContentDisposition();
+    test:assertEquals(disposition.toString(), "inline", msg = "Found unexpected output");
+}
+
+
```
Comment (chamil321) [inline mime-ballerina/tests/mime_test.bal:1016]:
```suggestion
```

File: mime-ballerina/tests/mime_test.bal [hunk 7]
```diff
@@ -95,6 +112,21 @@ public function testGetContentDispositionObject() {
...
+    entity.setHeader(CONTENT_DISPOSITION, ";");
+    var result = trap entity.getContentDisposition();
+
+    if result is error {
+        test:assertEquals(result.message(), "invalid header value: ;",
+            msg = "Found unexpected output");
+    } else {
+        test:assertFail(msg = "Found unexpected output type");
+    }
+
```
Comment (chamil321) [inline mime-ballerina/tests/mime_test.bal:127]:
```suggestion
```

**Comment by Dilhasha** [comment]
@TharmiganK Let's use imperative form in commit messages. Please refer to below article on writing commit messages.
https://chris.beams.io/posts/git-commit/

**Comment by TharmiganK** [comment]
> @TharmiganK Let's use imperative form in commit messages. Please refer to below article on writing commit messages.
> https://chris.beams.io/posts/git-commit/

Sure, Thanks!

**Review by chamil321 — APPROVED** [review]

**Pull request merged**